docs/articles/semantic_search_news.md

```diff
@@ -0,0 +1,376 @@
+# Semantic search in business news - a notebook article
+
+Semantic search is revolutionizing how we discover and consume news articles, offering a more intuitive and efficient method for finding relevant content and curating personalized news feeds. By embedding the nuances and underlying concepts of text documents in vectors, we can retrieve articles that align closely with the user's interests, preferences, and browsing history.
+
+Still, implementing effective semantic search for news articles presents **challenges**, including:
+
+- **Response optimization**: you need to figure out how to weight data attributes in your semantic search algorithms
+- **Scalability and performance**: you need efficient indexing and retrieval mechanisms to handle the vast volume of news articles
+
+Superlinked is designed to handle these challenges, empowering you to **scale efficiently** and - using Superlinked Spaces - **prioritize text similarity and/or recency so you can recommend highly relevant news articles to your users *without having to re-embed your dataset*.** 
+
+To illustrate, we'll take you step by step through building a **semantic-search-powered business news recommendation app**, using the following parts of Superlinked's library:
+
+- **Recency space** - to encode the recency of a data point
```

Comment:
would be nice to link respective feature notebooks maybe?
https://github.com/superlinked/superlinked/blob/main/notebook/feature/recency_embedding.ipynb
https://github.com/superlinked/superlinked/blob/main/notebook/feature/text_embedding.ipynb
https://github.com/superlinked/superlinked/blob/main/notebook/feature/query_time_weights.ipynb